### DISCUSSION TITLE
Appreciation

### DISCUSSION THREAD

**Discussion by Kristianfriis** [Dreamescaper/BlazorBindings.Maui#116] [General]
Just a quick appreciation for the work that you have done. 
I have always found XAML too verbose and unwieldy in many aspects. Especially data binding just does not make sense to me. 
I am a big fan of Blazor, but did not want to make a native using webview. I wanted native controls and blazor bindings. 
And to my great surprise the Experimental Mobile Blazor Bindings are being carried on by you all. 

So just a big thank you for making Blazor in MAUI a possibility

**Comment by Dreamescaper** [comment]
Thanks a lot for kind words!
Please let me know if you have any questions or suggestions.

**Comment by Kristianfriis** [comment]
Will do. I am looking through the implementations, so I get a feeling for it. 
At some point when it all makes more sense to me, I will gladly contribute, since this is the main way I would like to create native app :-)

**Comment by Laftek** [comment]
I would like to express my big thanks as well 🙏 you are doing great job and whole MBB should have much higher priority because this could be the main way for .NET to create native apps. Thanks again and keep it going! We all apprieciate your work.
